.env.example

```diff
@@ -14,7 +14,7 @@ VITE_RGS="https://rgs.mutinynet.com/snapshot/"
 VITE_SELFHOSTED="true"
 VITE_AUTH="https://auth-staging.mutinywallet.com"
 VITE_SUBSCRIPTIONS="https://subscriptions-staging.mutinywallet.com"
-VITE_STORAGE="https://storage-staging.mutinywallet.com"
+VITE_STORAGE="https://storage-staging.mutinywallet.com/v2"
```

Comment:
just double checking, is this the only ENV change that needs to be added to `.env.example` ?

Reply:
yeah, nothing else changed